filament/backend/src/metal/MetalContext.h

```diff
@@ -53,6 +53,17 @@ struct MetalContext {
     id<MTLCommandBuffer> currentCommandBuffer = nullptr;
     id<MTLRenderCommandEncoder> currentCommandEncoder = nullptr;
 
+    // This flag is set to true iff a FrameFinishedCallback is passed to beginFrame().
```

Comment:
typo `iff` -> `if`

Reply:
So I was using `iff` to mean "if and only if", but maybe I should write it out.